index.html

```diff
@@ -799,14 +799,14 @@ <h4>
     </p>
     <pre class="highlight">
 const reader = new NFCReader();
-reader.scan({ url: "https://mygame.com/mypath/mygame" });
+reader.scan({ id: "https://mygame.com/mypath/mygame" });
```

Comment:
Just to understand it: `id` is an addition, and not a replacement for `url`, but in this example we test a URL as the `id`. Then, an app can specify any `id` when creating content? 
I think this is why I asked about the name, since NDEF message is an NDEF level thing, but message identifier is an app-level concept. The two need to be mapped to each other but now a casual reader may confuse them. Provided I understand the change correctly.

Reply:
- "id" is an addition
- according to NDEF spec the generator (say NFCWriter) is responsible for the uniqueness of the id's meaning we enforce the same restrictions here as we do for the author record.
- the idea is to remove url and author record after this